applications/GeoMechanicsApplication/tests/fixed_spatial_variation/simple_mesh.mdpa

```diff
@@ -0,0 +1,104 @@
+Begin Table 1 X WATER_PRESSURE
+    0.0000000000 0.0000000000
+    1.0000000000 -5000
+End Table
+
+Begin Table 2 X WATER_PRESSURE
+    0.0000000000 -40000
+    1.0000000000 -30000
+End Table
+
+Begin Properties 1
+End Properties
+
+
+Begin Nodes
+  1  1.0000000000 -2.0000000000 0.0000000000
+  2  0.0000000000 -2.0000000000 0.0000000000
+  4  1.0000000000 0.0000000000 0.0000000000
+  5  0.0000000000 0.0000000000 0.0000000000
+  6  0.5000000000 -2.000000000 0.0000000000
+  9  0.0000000000 -1.000000000 0.0000000000
+ 10  1.0000000000 -1.000000000 0.0000000000
+ 13  0.5000000000 0.0000000000 0.0000000000
```

Comment:
Interesting node IDs ;-) I would have expected a consecutive sequence of numbers, although what is written here should work perfectly fine. Just less intuitive, that's all.

Reply:
Changed the order to make it a bit easier to read